cmd/infracost/configure.go

```diff
@@ -5,17 +5,19 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/infracost/infracost/internal/config"
-	"github.com/infracost/infracost/internal/ui"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+
+	"github.com/infracost/infracost/internal/config"
+	"github.com/infracost/infracost/internal/ui"
 )
 
 var supportedConfigureKeys = map[string]struct{}{
 	"api_key":                  {},
 	"currency":                 {},
 	"pricing_api_endpoint":     {},
 	"enable_dashboard":         {},
+	"disable_hcl":              {},
```

Comment:
I've added this into the configuration file but have neglected to add it to the examples `--help` text as I thought it might be more confusing to users - considering we're adding this configuration to essentially support "soft" removal of the feature.